mws/apis/products.py

```diff
@@ -14,8 +15,8 @@
 class Products(MWS):
     """Amazon MWS Products API
 
-    Docs:
-    http://docs.developer.amazonservices.com/en_US/products/Products_Overview.html
+    `MWS Docs: Products API Overview
+    <https://docs.developer.amazonservices.com/en_US/products/Products_Overview.html>`_
```

Comment:
This change is also relevant to documentation. The docstring is displayed automatically in the docs with autodoc applied, so the links are adjusted to a style that makes sense in Sphinx, rather than previously when links made sense in code only.